### ISSUE TITLE
Export api objects per endpoints 

### ISSUE THREAD

**Issue by matamatanot** [aspidajs/aspida#468]
## Description

Export api objects per endpoints to reduce JS bundle size.

## Describe the solution you'd like

The more endpoints you have, **the larger** the bundle size will be.

Exmaple Repo : https://github.com/matamatanot/aspida-test

```
$api.ts

/* eslint-disable */
import { AspidaClient, BasicHeaders } from "aspida";
import { Methods as Methods0 } from "./pet/index";
import { Methods as Methods1 } from "./pet/_petId@number/index";
import { Methods as Methods2 } from "./pet/_petId@number/uploadImage";
import { Methods as Methods3 } from "./pet/findByStatus";
import { Methods as Methods4 } from "./store/inventory";
import { Methods as Methods5 } from "./store/order/index";
import { Methods as Methods6 } from "./store/order/_orderId@number";
import { Methods as Methods7 } from "./user/index";
import { Methods as Methods8 } from "./user/_username@string";
import { Methods as Methods9 } from "./user/createWithArray";
import { Methods as Methods10 } from "./user/createWithList";
import { Methods as Methods11 } from "./user/login";

const api = <T>(client: AspidaClient<T>) => {//}
export type ApiInstance = ReturnType<typeof api>;
export default api;
```

```
test1.tsx (use pet only)

import aspida from "@aspida/axios";
import api from "../apis/$api";
(async () => {
  const client = api(aspida());

  const pets = await client.pet.findByStatus.get({
    query: { status: ["available"] },
  });
  console.log(pets);
})();

```

Export Pet only
```
$api2.ts

/* eslint-disable */
import { AspidaClient, BasicHeaders } from "aspida";
import { Methods as Methods0 } from "./pet/index";
import { Methods as Methods1 } from "./pet/_petId@number/index";
import { Methods as Methods2 } from "./pet/_petId@number/uploadImage";
import { Methods as Methods3 } from "./pet/findByStatus";

const api = <T>(client: AspidaClient<T>) => {//}
export type ApiInstance = ReturnType<typeof api>;
export default api;
```

```
test2.tsx

import aspida from "@aspida/axios";
import api from "../apis/$api2";
(async () => {
  const client = api(aspida());

  const pets = await client.pet.findByStatus.get({
    query: { status: ["available"] },
  });
  console.log(pets);
})();

```

**Bundle Size**
<img width="860" alt="bundle size" src="https://user-images.githubusercontent.com/39780486/85403598-9e142c00-b598-11ea-849d-64a961462b3e.png">

## Describe alternatives you've considered
Declare another variable using api object.

## Additional context
There are endpoints that are no longer used but remain in the documentation(OpenAPI) in some environment.
In order for webpack and others bundlers to process them as a Dead Code, Export api objects as a separate variable.

**Comment by solufa** [comment]
@matamatanot 
Thank you for posting the Issue!
Tree Shaking support is an obvious need.
I'll figure out how to implement it.

**Comment by matamatanot** [comment]
@solufa 
Thanks! 
For reducing bundle size, there is room for other improvements.

**Export only `get` or `$get` or both (now default) methods.**
Some users use one or the other, not both.
It is preferable to have the option to choose. Users can decide which one to use from cli option.

The same goes for the `post` method.

**Comment by solufa** [comment]
@matamatanot 
I certainly don't use both very often.
I'll consider adding cli option.

**Comment by solufa** [comment]
@matamatanot 
Added `outputEachDir` option to generate `$api.ts` per endpoint.
openapi2aspida has been updated too.
See README
https://github.com/aspidajs/aspida#import-only-some-endpoints

I didn't add the option to limit the method output to one side because of the conflicts with other products.
Instead, I've optimized `$api.ts` for easy compression with uglify
- Delete async/await
- Assign the common part of a string to a variable

It's not verified, but it should be significantly more compressed than before.
Thank you for helping to improve aspida!
